### DISCUSSION TITLE
Please help me with this issue,I want to know why this failed? #1613

### DISCUSSION THREAD

**Discussion by shivambhat** [EddieHubCommunity/support#302] [Q&A] [answered]
Please help me with this issue, I want to know why this failed? 

Link: https://github.com/OpenSouceCode/Hacktoberfest2020/issues

**Comment by eddiejaoude** [comment, marked as answer]
@shivambhat that is the list of issues. Please can you explain which issue you are referring to and what failed?

**Reply by shivambhat** [reply]
> @shivambhat that is the list of issues. Please can you explain which issue you are referring to and what failed?

It was sorted, Thanks a lot
